addons/info.go

```diff
@@ -0,0 +1,50 @@
+package addons
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/db"
+)
+
+func Info(app, addon string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+	addonInfo, err := c.AddonShow(app, addon)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
+	}
+
+	dbInfo, err := db.Show(app,addon)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
+	}
+
+	forceSsl, internetAccess := "disabled", "disabled"
+	if featuresLen := len(dbInfo.Features); featuresLen > 0 {
+		if featuresLen == 2 {
```

Comment:
This is not really true. There is a possibility that length is greater than 2 and that internet access is enabled.

Reply:
Do you know where I can find the list of possible features for a db ? Thought they were only two.

Reply:
We don't maintain a comprehensive list of features. Do you have access to appsdeck-database code?

Reply:
I don't think so.

Reply:
OK so you don't have access to the full list of features.. 😓